### PR TITLE
Stop ignoring ControlStructures.EarlyExit

### PR DESCRIPTION
**Pull request by malarzm**
Automatic fixes

### CODE REVIEW

File: lib/Doctrine/ODM/MongoDB/Mapping/ClassMetadata.php [hunk 1]
```diff
@@ -845,7 +845,9 @@ public function addIndex($keys, array $options = [])
                     $lower = strtolower($value);
                     if ($lower === 'asc') {
                         return 1;
-                    } elseif ($lower === 'desc') {
+                    }
+
+                    if ($lower === 'desc') {
```
Comment (malarzm) [inline lib/Doctrine/ODM/MongoDB/Mapping/ClassMetadata.php]: I'm not sure why it was caught (unless `elseif` is a bad practice), there's also one more such change. Should I revert?

Comment (alcaeus) [reply lib/Doctrine/ODM/MongoDB/Mapping/ClassMetadata.php]: That looks like a bug. We should change them back and report it upstream.

File: lib/Doctrine/ODM/MongoDB/Mapping/ClassMetadata.php [hunk 2]
```diff
@@ -1703,9 +1709,11 @@ public function setParentClasses(array $classNames)
     {
         $this->parentClasses = $classNames;
 
-        if (count($classNames) > 0) {
-            $this->rootDocumentName = array_pop($classNames);
+        if (count($classNames) <= 0) {
```
Comment (Majkl578) [inline lib/Doctrine/ODM/MongoDB/Mapping/ClassMetadata.php]: This can never be <0, but I understand how this happened. :D

Comment (malarzm) [reply lib/Doctrine/ODM/MongoDB/Mapping/ClassMetadata.php]: Better safe than sorry... or something :P

**Pull request merged**